tests/contrib/minikube/test_kubernetes_pod_operator.py

```diff
@@ -111,6 +111,21 @@ def test_delete_operator_pod():
         )
         k.execute(None)
 
+    def test_keep_failed_pod(self):
+        k = KubernetesPodOperator(
+            namespace='default',
+            image="ubuntu:16.04",
+            cmds=["bash", "-cx"],
+            arguments=["exit 1"],
+            labels={"foo": "bar"},
+            name="test",
+            task_id="task",
+            is_delete_operator_pod=True,
+            keep_failed_pod=True
+        )
+        with self.assertRaises(AirflowException):
+            k.execute(None)
```

Comment:
Seems this test can't really test what happens when `keep_failed_pod=True`? It's only testing `arguments`.

Reply:
many thanks fo reviewing the PR, those comments all make sense.

That's a fair comment about this test. Would sleeping for e.g. 10 seconds and then asserting the pod is still in the namespace and in a failed state be sufficient to test it?

Reply:
I'd like to avoid sleep in tests if we can (but it might not be avoidable) - does the pod enter a "deleting" state instantly? If so could we test that `not pod_exists or pod_in_deleteing_state`?